server/src/app.ts

```diff
@@ -0,0 +1,7 @@
+import express, { Request, Response, Application } from 'express';
+const app: Application = express();
```

Comment:
I would rename `app` to something less generic like `mainNode` or `coordinatorNode` . Then the new components would take the naming convention of `<module>Node`.  You could also move this file to a directory named `nodes`.

Reply:
Probably we will not use only node.js nodes, because the purpose of this project is to use different technologies (e.g. python)

Reply:
Good point.  We could just add more directories for different technologies as they are integrated, no?

Reply:
yes, we can add more directories. But directory name will come out in the wash.